### DISCUSSION TITLE
Please make the docs easier to understand

### DISCUSSION THREAD

**Discussion by ClemensK** [formfcw/directus-extension-flexible-editor#54] [Feature Request]
Wanted to try using it with relationship nodes, but the doc is almost impossible to understand. The screencast is a fever dream. Please slow down! The documentary starts in the middle of nowhere.

**Comment by formfcw** [comment]
@ClemensK Thank you for your honesty. Is there anything in particular that you don't understand that I can help you with? How do you think it could be improved?

**Reply by ClemensK** [reply]
Thank you for your very quick response! 
I think it would be great if the screencast started earlier and explained the structure of **pages** and **editor nodes** in more detail. Maybe just make an example from scratch.

**Reply by formfcw** [reply]
Would any of these videos help you?


https://github.com/user-attachments/assets/12433b29-1044-481e-a9ea-8eda7c713a5d


https://github.com/user-attachments/assets/eee5c875-54b6-4d2f-ba04-1357c3010a05
